galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/sdv/AbstractSDVLocal.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.inttests.sdv;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.gson.JsonObject;
+
+import dev.galasa.BeforeClass;
+import dev.galasa.Test;
+import dev.galasa.cicsts.CicsRegion;
+import dev.galasa.cicsts.CicstsManagerException;
+import dev.galasa.cicsts.ICicsRegion;
+import dev.galasa.galasaecosystem.GalasaEcosystemManagerException;
+import dev.galasa.galasaecosystem.IGenericEcosystem;
+
+public abstract class AbstractSDVLocal {
+
+  @CicsRegion(cicsTag = "A")
+  public ICicsRegion cics;
+  
+  @BeforeClass
+    public void setProperties() throws GalasaEcosystemManagerException, CicstsManagerException {
+    // Setting these properties in the shadow ecosystem
+    getEcosystem().setCpsProperty("cicsts.provision.type", "DSE");
+    getEcosystem().setCpsProperty("cicsts.dse.tag.A.applid", cics.getApplid());
+    getEcosystem().setCpsProperty("cicsts.dse.tag.A.version", cics.getVersion().toString());
+    getEcosystem().setCpsProperty("cicsts.default.logon.initial.text", "HIT ENTER FOR LATEST STATUS");
+    getEcosystem().setCpsProperty("cicsts.default.logon.gm.text", "******\\(R)");
+
+    // SDV CPS props
+    getEcosystem().setCpsProperty("sdv.roleTag.R1.role", "TELLER");
+    getEcosystem().setCpsProperty("sdv.zosImage." + cics.getZosImage().getImageID() + ".role.TELLER.credTags", "???");
+    getEcosystem().setCpsProperty("sdv.cicsTag.A.port", "???");
+    getEcosystem().setCpsProperty("sdv.cicsTag.A.hlq", "???");
```

Comment:
Hi @jadecarino, these are the CPS properties I've trouble filling in for the SDV IVT. 

I need a credentials tag, probably the one for the user running the test would do...
A port number for the SDC service to run on
and a HLQ for the CICS install.....not sure if `ANTZ.CICS.TS.DEV.INTEGRAT` would be safe to put here?

Reply:
another thing to consider is....do we want to make these values public, in this file?

Reply:
@Mark-J-Lawrence Hi Mark. So just to clarify, do these CPS properties for your manager definitely have to be set in the child ecosystem that's spawned to run the IVT, or could they go just in our prod Galasa ecosystem's CPS properties?

Reply:
@jadecarino No they definitely don't, if they already exist as part of the ecosystems CPS properties, I can remove them from here